modules/profiles/laptop-x86.nix

```diff
@@ -112,6 +112,11 @@ in {
       profiles = {
         applications.enable = false;
       };
+
+      # Logging services
+      logging.client.enable = true;
+      logging.server.enable = true;
+      logging.server.address = "ghaflogs.vedenemo.dev";
```

Comment:
This is a bit confusing. Is this running in all VMs, just the host or both?

The log collection bits should be configured in a profile that gets imported in all VMs and the host, and the log processing thing in a profile that only gets imported on the admin-vm.